src/libraries/Native/AnyOS/System.IO.Compression.Native/entrypoints.c

```diff
@@ -25,6 +28,23 @@ static const Entry s_compressionNative[] =
     DllImportEntry(BrotliEncoderDestroyInstance)
     DllImportEntry(BrotliEncoderHasMoreOutput)
     DllImportEntry(BrotliEncoderSetParameter)
+    DllImportEntry(BrotliGetTransforms)
+    DllImportEntry(BrotliDecoderSetParameter)
+    DllImportEntry(BrotliGetDictionary)
+    DllImportEntry(BrotliDecoderTakeOutput)
+    DllImportEntry(BrotliEncoderIsFinished)
+    DllImportEntry(BrotliDecoderVersion)
+    DllImportEntry(BrotliDecoderIsUsed)
+    DllImportEntry(BrotliSetDictionaryData)
+    DllImportEntry(BrotliDecoderGetErrorCode)
+    DllImportEntry(BrotliEncoderTakeOutput)
+    DllImportEntry(BrotliEncoderVersion)
```

Comment:
Why do we need to export these? They do not seem to be used anywhere in the .cs files.

Reply:
They are exported and the diffing script found them. I am not sure why we export them.

Reply:
We should not be exporting them if we are not using them. It just keeps dead unused code around.